install.sh

```diff
@@ -1,11 +1,31 @@
 #!/bin/bash
 
-SSID="${1:-WiFiAPi}"
-PASSPHRASE="${2:-Raspberrypi}"
-IP_RANGE="${3:-192.168.1}"
+SSID="${1:-RaspberryPi}"
+PASSPHRASE="${2:-raspberry}"
+IP_RANGE="${3:-192.168.222}"
 INC="${4:-eth0}"
 WIFI="${5:-wlan0}"
 
+pause() {
+#	read -p "Appuyer sur une touche pour continuer ..." var
```

Comment:
No comments in other languages. Please remove.

Reply:
I will correct that, of course.

Reply:
The sentence is now in English with the new pull request. I also added a comment in the script to clarify.